glyphs2fontir/src/source.rs

```diff
@@ -3014,4 +3014,10 @@ mod tests {
             [Tag::new(b"wght")]
         );
     }
+    // this font causes a crash when we try to generate the notdef glyph, due to
+    // funny phantom locations?
```

Comment:
the test name hmm and the comment that suggests we aren't sure what's happening both inspire doubt in the reader. Can we make it a bit more confident about itself?